lib/ClientTlsContext.php

```diff
@@ -83,7 +83,7 @@ public function withCiphers(string $ciphers = null): self {
         return $clone;
     }
 
-    public function getCiphers(): string {
+    public function getCiphers() {
```

Comment:
Why that?

Reply:
Because the default value is `null`. First I tried to change that to be an empty string, but then other tests broke.

Reply:
That's a bug, indeed. It should return `$this->ciphers ?? \OPENSSL_DEFAULT_STREAM_CIPHERS`.

https://github.com/amphp/socket/blob/001aef1565583dda95bb94530e1b6644987fda66/lib/ClientTlsContext.php#L81

Reply:
The `?? \OPENSSL_DEFAULT_STREAM_CIPHERS` in `withCiphers` can then safely be removed.